webpack.config.js

```diff
@@ -1,8 +1,9 @@
 var webpack = require('webpack');
 var development = process.env.NODE_ENV !== "production";
+var path = require('path');
 
 var PATHS = {
-  BUILD: __dirname + '/dist',
+  BUILD: path.join(__dirname, '..', 'example_site/public'),
```

Comment:
This is dangerous in that your referencing a folder outside of the scope of this code repo, you should use a symlink in your example_site instead and keep this to `/dist`